### DISCUSSION TITLE
No window for Battery Toolkit.

### DISCUSSION THREAD

**Discussion by GleoIT** [mhaeuser/Battery-Toolkit#61] [Q&A] [answered]
I don't get a window for Battery Toolkit after installing it and running it. Just shows in my Dock. First installed it with ctrl + open, in my Downloads folder and also tried dragging it to Applications and running it. Allowed background activity when asked. Did this with macbook air m2 and originally on Sonoma 14.7.? and now on Sequoia 15.2. Any idea what the problem might be?

Also, really appreciate you sharing this software. Exactly what I was looking for.

**Comment by mhaeuser** [comment, marked as answer]
There is no “main window” as it’s background application. You can open Settings from the application menu or the menu bar extra.
